experience.html

```diff
@@ -0,0 +1,103 @@
+            Lived here a majority of my life. Food here was definitely top tier
+            but there was one place in the suburbs that had the best chicken
+            I've had in my entire life and no other place I've been to has been
+            like it. Lots of events and the people are ok. Like a mix of NYC
+            rudeness and southern hospitality. There's also this one guy who
+            stands outside a stadium and just condems peple all day who has his
+            own wiki page so that's something. 8/10
+          </p>
+          <img src="img/chicago.jpg" alt="chicago" />
+          <!-- </div> -->
+          <!-- <div class="dmv"> -->
+          <p>DMV</p>
+          <p>
+            The DMV, mainly Nothern Virginia, was interesting. First of all, the
+            public transit there was immmaculate in my opinion. Chicago's was
+            good but something about the effiency of the transit in DMV was very
+            satisfying to me. The wealth disparity was also a lot more apparent.
+            It was either luxury living or not so great living conditions.
+            Didn't see much of anything in the middle. The food was good.
+            Nothing crazy memborable about the food other than the seafood
+            tasting fresher. Skipped out on the dmv brunch scene so that's going
+            to skew this review a bit too. Overall, 7/10 BUT I do think if I go
+            back and experience it now it'll be an 8 or 9 out of 10 just because
+            the weather there was solid. Never too cold or too hot.
+          </p>
+          <img src="img/dc.jpg" alt="DC" />
+          <!-- </div> -->
+          <!-- <div class="cin"> -->
+          <p>Cincinnati</p>
+          <p>
+            Errrrmmmmmm Cincinnati Ohio felt like a fever dream to me. There was
+            this faint smell of pee in the downtown area no matter where I went
+            and it was relatively dirty compared to its size. They had this free
+            trolley that went around the city so that was a plus. Food was just
+            normal. They're apparently known for their chili but I've had canned
+            chili better than their famous stuff. Would I go back here? No. The
+            only positive thing I can think of is that it's a cheaper place to
+            live in and the traffic was never crazy. 5/10
+          </p>
+          <img src="img/cin.jpg" alt="cincinnati" />
+          <!-- </div> -->
+          <!-- <div class="spring"> -->
+          <p>Springfield Illinois</p>
+          <p>
+            3/10. Felt like very little changed about the town since Abe Lincoln
+            died. It was like stepping back in time and not in a good way. It's
+            a small town so i wasn't expecting much and even then I was still
+            disappointed.
+          </p>
+          <img src="img/spring.jpg" alt="Springfield" />
+          <!-- </div> -->
+          <!-- <div class="tulsa"> -->
+          <p>Tulsa OK</p>
```

Comment:
Example here with p tag used for Tulsa, OK - is this best identified as a paragraph?  IS there another tag that could be used here to better identify the purpose of this element?